js/helper.js

```diff
@@ -282,11 +282,17 @@ function getPath() {
  * @param {string[]} args Command line arguments passed to `sentry-cli`.
  * @param {boolean} live We inherit stdio to display `sentry-cli` output directly.
  * @param {boolean} silent Disable stdout for silents build (CI/Webpack Stats, ...)
+ * @param {boolean} silentLogs Show only errors and success messages, hide verbose logs
  * @param {string} [configFile] Relative or absolute path to the configuration file.
  * @param {Object} [config] More configuration to pass to the CLI
  * @returns {Promise.<string>} A promise that resolves to the standard output.
  */
-async function execute(args, live, silent, configFile, config = {}) {
+async function execute(args, live, silent, silentLogs, configFile, config = {}) {
```

Comment:
[change request] Unless I am missing something, I think changing the function signature in this way will be a breaking API change. Is that correct?

If so, please make the `silentLogs` parameter optional, so the change is backwards compatible (or propose some other way to do this, I am not super familiar with JS idioms).